static_frame/core/util.py

```diff
@@ -3299,13 +3295,13 @@ class PositionsAllocator:
     '''Resource for re-using a single array of contiguous ascending integers for common applications in IndexBase.
     '''
     _size: int = 1024 # 1048576
-    _array: NDArrayAny = np.arange(_size, dtype=DTYPE_INT_DEFAULT)
+    _array: TNDArrayAny = np.arange(_size, dtype=DTYPE_INT_DEFAULT)
```

Comment:
How come not `np.ndarray[np.int_]`?

Reply:
Same for all other functions on this class

Reply:
Good catch!